services/Booking.json

```diff
@@ -0,0 +1,15 @@
+{

```

Comment:
This services has not consistent filtered content when fetched twice in a row (the date time at the beginning of the doc changes at each request), which means it will result in false positive notifications. 
So this date time element should be removed. For example, remove the paragraph child next to the print button `"remove": "a.print-button, .print-button + p"`

Reply:
To locally validate services you have modified, you can use the command `npm run validate:modified`.
Also, if you want to validate a specific service you can use the command `npm run validate Booking` for example